### PR TITLE
fix(container): update sidebar

### PR DESCRIPTION
**Pull request by cbourgois**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/product-nav-reshuffle`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix MANAGER-9368,
| License          | BSD 3-Clause

## Description

Update sidebar.

Commits should be squashed.

### CODE REVIEW

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/navigation-tree/root.ts [hunk 1]
```diff
@@ -1,34 +1,94 @@
...
-      translation: 'sidebar_home',
+      id: 'sunrise',
+      translation: 'sidebar_sunrise',
       routing: {
-        application: 'hub',
+        application: 'sunrise',
       },
       count: false,
+      feature: 'sunrise',
+      region: ['EU', 'CA'],
```
Comment (marie-j) [inline packages/manager/apps/container/src/container/nav-reshuffle/sidebar/navigation-tree/root.ts:30]: Why not feature flipp this ?

Comment (cbourgois) [reply packages/manager/apps/container/src/container/nav-reshuffle/sidebar/navigation-tree/root.ts:31]: we feature-flip it :) Forgot to remove old `region` property.

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/style.module.scss [hunk 2]
```diff
@@ -8,9 +8,14 @@
   height: 100%;
   display: flex;
   flex-direction: column;
+  justify-content: flex-end;
   min-width: 20rem;
   color: white;
 
+  &_menu {
+    // overflow: scroll;
```
Comment (zakariasahmane) [inline packages/manager/apps/container/src/container/nav-reshuffle/sidebar/style.module.scss:16]: Should not be commented, if it is, sidebar won't work on smaller screen sizes when there are a lot of elements and Assistance is Open

We can see the issue in the staging

Comment (cbourgois) [reply packages/manager/apps/container/src/container/nav-reshuffle/sidebar/style.module.scss:16]: It breaks the display in some case. We will discuss about it and provide a solution.

**Review by zakariasahmane — APPROVED** [review]

**Review by Jisay — APPROVED** [review]

**Pull request merged**